src/Uno.UI/UI/Xaml/Controls/TimePicker/NativeTimePickerFlyout.Android.cs

```diff
@@ -58,7 +74,7 @@ private void AdjustAndSaveTime(int hourOfDay, int minutes)
 
 	private void ShowTimePicker()
 	{
-		var time = Time.RoundToNextMinuteInterval(MinuteIncrement);
+		var time = _initialTime.RoundToNextMinuteInterval(MinuteIncrement);
```

Comment:
If time is set programmatically, will it still work?

Reply:
It should, If Time is set programmatically before the flyout is opened, _initialTime is updated accordingly through the `SaveInitialTime` call in Open.